### DISCUSSION TITLE
Личка в Instagram

### DISCUSSION THREAD

**Discussion by ser9ey01** [dovecoteescapee/ByeDPIAndroid#140] [General]
Благодарю за приложение! Инстаграм работает хорошо, кроме лички. Почему-то при заходе в личные сообщения там ничего не обновляется. Если заходить под другим VPN я вижу новые сообщения, но при ByeDPI личные сообщения не обновляются. Куда можно копать?

**Comment by zimbabwe303** [comment]
Instagram, Facebook, Twitter и ещё многие сайты заблокированы по IP, полностью или частично (зависит от проавйдера) . Все старые блокировки, сделанные ещё до повсеместного внедрения ТСПУ, не обходятся такими методами как ByeDPI. Вообще это, скорее, остроумный хак, чем основательное решение, пройдёт сколько-то времени и его научатся глушить полностью. С другой стороны, наш остроумный народ тоже думает и сопротивляется. Что-нибудь да придумается. Да и китайские товарищи по несчастью в помощь - вон уже сколько всего придумали: [Xray](https://github.com/XTLS/Xray-core) с VLESS-XTLS-uTLS-REALITY, [sing-box](https://github.com/SagerNet/sing-box) с Shadowsocks+AEAD 2022, [Cloak](https://github.com/cbeuw/Cloak) и ещё вагон (но нужен VPS и умение это всё хозяйство настроить).
